### PR TITLE
draft-js #120

### PR DESCRIPTION
**Pull request by balthazzar**
https://github.com/webRunes/Core-WRIO-App/issues/120

Add link recognition
fix saveAs
fix keys generator

### CODE REVIEW

**Comment by AlexeyAnshakov** [comment]
@michbil please review

File: src/client/js/client.js [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+    applyMentions
+}
+from './mixins/mentions';
 import getHttp from './getHttp';
 import CoreEditor from './CoreEditor';
-import {ContentBlock, CharacterMetadata} from 'draft-js';
+import {
+    ContentBlock, CharacterMetadata
+}
+from 'draft-js';
```
Comment (michbil) [inline src/client/js/client.js]: Stylistic: make single line

File: src/client/js/CoreEditor.js [hunk 2]
```diff
@@ -295,7 +327,9 @@ var INLINE_STYLES = [{
 }, ];
 
 const InlineStyleControls = (props) => {
-    let {editorState} = props;
+    let {
```
Comment (michbil) [inline src/client/js/CoreEditor.js]: @balthazzar make single-line too

**Comment by balthazzar** [comment]
@michbil done

**Pull request merged**